styles/components/_form.scss

```diff
@@ -9,6 +10,14 @@ form {
     display: flex;
     align-items: center;
   }
+
+  &.compact label {
+    line-height: $formElementHeight * 0.75;
+  }
+
+  &.veryCompact label {
+    line-height: $formElementHeight * 0.5;
+  }
```

Comment:
The compact and veryCompact classes are for input elements that should have form styling, but not as much vertical space.